cmd/balena-engine/main.go

```diff
@@ -23,7 +23,7 @@ func main() {
 	command := filepath.Base(os.Args[0])
 
 	switch command {
-	case "balena":
+	case "balenadctl":
```

Comment:
Since all those binaries are not directly used by users when interacting with balena I think we should namespace all of them under `balena-engine`, so have `balena-engine-containerd` etc. Regarding the client name I'm still toying with the idea of having `balena-engine` as the CLI name instead of `balenadctl`. On the `balena-daemon` thing, I'm up for also namespacing it under `balena-engine-daemon`

Reply:
Done:

From | To
--- | ---
balena | balena-engine
balenad | balena-engine-daemon
balena-containerd | balena-engine-containerd
balena-containerd-shim | balena-engine-containerd-shim
balena-containerd-ctr | balena-engine-containerd-ctr
balena-runc | balena-engine-runc
balena-proxy | balena-engine-proxy
balena-init | balena-engine-init